.github/workflows/documentation-links.yml

```diff
@@ -0,0 +1,16 @@
+name: readthedocs/actions
+on:
+  pull_request_target:
+    types:
+      - opened
```

Comment:
This is also different from the `docs` workflow

Reply:
Seems to result in a skipped job =/ https://github.com/lab-cosmo/equistore/actions/runs/4343538892/jobs/7585704675